doc/panes/editor.rst

```diff
@@ -27,35 +27,273 @@ The Editor pane consists of the following areas:
 6. The tab bar displays the names of all opened files.
    It also has a :guilabel:`Browse tabs` button (at left) to show every open tab and switch between them—which comes in handy if many are open.
 
+=========
+Interface
+=========
 
+----
+Tabs
+----
 
-===================
-Defining code cells
-===================
+You can browse and navigate between open files in the Editor with tabs.
+Click the :guilabel:`Browse tabs` button on the left of the Tabs bar to display a list of open files, with the active tab checked.
+Reorder files by dragging and dropping, or with :guilabel:`Sort tabs alphabetically` in the options menu, which also allows closing all tabs to the right of, or all tabs but the active one.
+
+.. image:: /images/editor/editor-tabs-browser.png
+   :alt: Spyder's Editor pane, showing the tabs browser
+
+-------------
+File switcher
+-------------
+
+The Editor features a file switcher, which enables you to navigate and switch between multiple open files.
+The file switcher is helpful for locating any file when there are several files opened.
+It can be accessed from the :menuselection:`File --> File Switcher` menu or :kbd:`Ctrl-P` and includes a search function.
+You can type in any part of an open file's name and -if exists- it can be switched to by pressing Enter.
+
+.. image:: /images/editor/editor-file-switcher.png
+   :alt: Spyder's Editor pane, showing the file switcher
+
+------------
+Split panels
+------------
+
+The Editor can be split horizontally and vertically into as many distinct panels as desired.
+This allows viewing and editing the contents of several files (or different parts of the same file) at the same time.
+
+GIF needed
+
+Split the Editor with the :guilabel:`Split vertically` (:kbd:`Ctrl-Shift-{`) and :guilabel:`Split horizontally` (:kbd:`Ctrl-Shift–`) commands in the options menu, and use :guilabel:`Close this panel` (:kbd:`Alt-Shift-W`) to close the selected split panel.
+.. note: :menuselection:`Close this panel` closes a split panel, while :menuselection:`Close` hides the entire Editor *pane* (including all splits, which are restored when the Editor is re-opened).
+
+
+================
+Editing features
+================
+
+-------------------
+Syntax highlighting
```

Comment:
This section is rendering as a whole paragraph, is it intended to look like that?

Reply:
@CAM-Gerlach No. Thanks